docs/examples/ipfs-101/README.md

```diff
@@ -0,0 +1,109 @@
+# IPFS 101, spawn a node and add a file to the IPFS network
+
+> This example is the sister example of IPFS 101 for [js-ipfs](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101).
+
```

Comment:
I'll update this README once I have `main.go` working. Currently this is just c&p from https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101